backend/PyMatcha/__init__.py

```diff
@@ -59,8 +59,6 @@
 if os.getenv("ENABLE_LOGGING") == "True":
     setup_logging()
 
-application = Flask(__name__, static_folder=os.getenv("FRONT_STATIC_FOLDER"))
```

Comment:
You cannot delete that, it's what creates the flask app. Just delete the `static_folder=os.getenv("FRONT_STATIC_FOLDER")`